components/modal/src/css/popup-modal.css

```diff
@@ -0,0 +1,112 @@
+@import url(../../../../css/component.css);
+
+.center,
+.popup-content {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  justify-content: center;
+  align-items: center;
+}
+
+#click {
+  display: none;
+}
```

Comment:
I would suggest to used class for display: none and name it accordingly. then you can even reuse it in future.